glide.yaml

```diff
@@ -53,6 +53,8 @@ import:
   - ssh
 - package: golang.org/x/oauth2
 - package: gopkg.in/cheggaaa/pb.v1
+- package: github.com/asaskevich/govalidator/
```

Comment:
Nice one. Seems to do what we want. Not that we could not have done this ourselves ;-)